tachyon/crypto/commitments/merkle_tree/field_merkle_tree/field_merkle_tree_mmcs.h

```diff
@@ -129,13 +140,8 @@ class FieldMerkleTreeMMCS final
           return IndexedDimensions{index, dimensions};
         });
 
-    // TODO(chokobole): Use https://github.com/timsort/cpp-TimSort or
-    // https://github.com/sebawild/powersort for better performance.
-    std::stable_sort(
-        sorted_dimensions_list.begin(), sorted_dimensions_list.end(),
-        [](const IndexedDimensions& a, const IndexedDimensions& b) {
-          return a.dimensions.height > b.dimensions.height;
-        });
+    base::StableSort(sorted_dimensions_list.begin(),
```

Comment:
Is using StableSort also a part of bug fix? If then, can we squash with the prior commit?

Reply:
The commit is a part of #492 and that's why I said this should be merged after #492.